examples/prometheus/prometheus.yaml

```diff
@@ -48,10 +53,12 @@ objects:
     name: prometheus
     namespace: "${NAMESPACE}"
   spec:
+    host: "${PROMETHEUS_HOSTNAME}"
     to:
       name: prometheus
     tls:
       termination: Reencrypt
+      destinationCACertificate: "${SERVICE_SIGNER_CERT}"
```

Comment:
I get a failure when I try to remove it:
The Route "prometheus" is invalid: spec.tls.destinationCACertificate: Required value